pkg/oci/remote/referrers.go

```diff
@@ -25,7 +25,9 @@ import (
 func Referrers(d name.Digest, artifactType string, opts ...Option) (*v1.IndexManifest, error) {
 	o := makeOptions(name.Repository{}, opts...)
 	rOpt := o.ROpt
-	rOpt = append(rOpt, remote.WithFilter("artifactType", artifactType))
+	if artifactType != "" {
```

Comment:
The `artifactType` for bundles is something like `application/vnd.dev.sigstore.bundle.v0.3+json`. Since the version number is baked into the type, we can't use the filter feature here. Therefore, `getBundles` uses an empty string as the artifactType in the call to Referrers.